src/io.c

```diff
@@ -129,15 +129,19 @@ avifIO * avifIOCreateFileReader(const char * filename)
     }
 
     fseek(f, 0, SEEK_END);
-    size_t fileSize = (size_t)ftell(f);
```

Comment:
ftell() returns -1 on failure. It would be bad to cast -1 to size_t or uint64_t. So I added error checking.

Strictly speaking, we should also check for fseek() failures, but I was too lazy to do that...